dts/bindings/can/nxp,canfd-flexcan.yaml

```
@@ -0,0 +1,25 @@
+# Copyright (c) 2022 Carl Zeiss Meditec AG
+# SPDX-License-Identifier: Apache-2.0
+
+description: NXP CANFD FlexCAN controller
+
+compatible: "nxp,canfd-flexcan"
```

```suggestion
compatible: "nxp,flexcan-fd"
```

I would suggest using `nxp,flexcan-fd` to better signal the relationship with the classic `nxp,flexcan` compatible. @dleach02 et al. Any preference here? The old one has the `nxp,kinetis-flexcan` compatible, but perhaps it ought to be renamed to just `nxp,flexcan`?

---

let me discuss internally on this point

---

Agree with using "flexcan-fs"

---

Fixed now.